packages/web/sentry.client.config.ts

```diff
@@ -0,0 +1,32 @@
+// This file configures the initialization of Sentry on the client.
+// The config you add here will be used whenever a users loads a page in their browser.
+// https://docs.sentry.io/platforms/javascript/guides/nextjs/
+
+import * as Sentry from "@sentry/nextjs";
+
+Sentry.init({
+  dsn: "https://c696452bb7ce4cc98150142ebea1c32f@o4505285755600896.ingest.sentry.io/4505285757698048",
+
+  // Adjust this value in production, or use tracesSampler for greater control
+  tracesSampleRate: 1,
+
+  // Setting this option to true will print useful information to the console while you're setting up Sentry.
+  debug: false,
+
+  replaysOnErrorSampleRate: 1.0,
+
+  // This sets the sample rate to be 10%. You may want this to be 100% while
+  // in development and sample at a lower rate in production
+  replaysSessionSampleRate: 0.1,
+
+  environment: process.env.NODE_ENV || "development",
```

Comment:
Perhaps we can differentiate between testnet envs? Via `IS_TESTNET` env var? Given the low volume of stage usage, I'm not concerned if an error gets captured in stage.

cc @MaxMillington

Reply:
Adding another env var for @daniel-farina to maintain in Vercel is a pain.